docs/data-sources/groups.md

```diff
@@ -24,9 +24,11 @@ data "twingate_groups" "foo" {
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
+### Optional
 
+- `is_active` (Boolean) Indicates if the Group is active
```

Comment:
Personally, I would use descriptions that indicate exactly how these are used to filter.
```
- `is_active` (Boolean) Returns only groups matching the specified state.
- `name` (String) Returns only groups that exactly match this name.
- `type` (String) Returns only groups of the specified type (valid: `MANUAL`, `SYNCED`, `SYSTEM`).
```

Reply:
@alexmensch what do you think?

Reply:
Yes agreed -- @vmanilo go ahead as above, just need to capitalize groups to `Groups`. Thanks all!

Reply:
fixed